kyo-caliban/src/main/scala/kyo/resolvers.scala

```diff
@@ -0,0 +1,65 @@
+package kyo
+
+import caliban.*
+import caliban.interop.tapir.*
+import caliban.interop.tapir.TapirAdapter.*
+import kyo.internal.KyoSttpMonad
+import sttp.tapir.Codec.JsonCodec
+import sttp.tapir.Endpoint
+import sttp.tapir.capabilities.NoStreams
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.netty.*
+import zio.*
+import zio.stream.ZStream
+
+opaque type Resolvers = Aborts[Throwable] & ZIOs
+
+object Resolvers:
+
+    private given StreamConstructor[Nothing] =
+        (_: ZStream[Any, Throwable, Byte]) => throw new Throwable("Streaming is not supported")
+
+    def run[T, S](v: HttpInterpreter[Any, CalibanError] < (Resolvers & S)): NettyKyoServerBinding < (ZIOs & Aborts[Throwable] & S) =
+        run[T, S](NettyKyoServer())(v)
+
+    def run[T, S](server: NettyKyoServer)(v: HttpInterpreter[Any, CalibanError] < (Resolvers & S))
+        : NettyKyoServerBinding < (ZIOs & Aborts[Throwable] & S) =
```

Comment:
Should it be `Aborts[CalibanError]` instead?

Reply:
That's right! I was hesitating whether I should expose it but yeah, why not use the most specific error type. Changed it.